R/as.SpatRaster.R

```diff
@@ -0,0 +1,57 @@
+#' as.SpatRaster
+#'
+#' Convert magclass object to a  object
```

Comment:
Did you leave some blanks for the reviewers to fill? :smile: 
```suggestion
#' Convert magclass object to a SpatRaster object. Requires the terra package.
```